invenio_communities/communities/schema.py

```diff
@@ -54,7 +54,7 @@ class CommunityMetadataSchema(Schema):
     ]
 
     title = SanitizedUnicode(required=True, validate=_not_blank(max=250))
-    description = SanitizedHTML(validate=_not_blank(max=2000))
+    description = SanitizedUnicode(validate=_not_blank(max=2000))
```

Comment:
The issue I see is that currently in Zenodo communities, `description` is a rich text field, and thus we would require to migrate the current `description` to some other field in order to preserve the original usage (cc @lnielsen). 

I can see that the purpose is to have an optional short tagline under the community's title, which differs from the original purpose... Let's discuss maybe what's the best design (or if we just take the toll when migrating)